package.json

```diff
@@ -99,8 +99,7 @@
     "admin-plugin-healthcheck": "^1.1.0",
     "amqplib": "^0.10.3",
     "amqplib-v0": "npm:amqplib@^0.8.0",
-    "apollo-server-express": "^2.24.1",
-    "apollo-server-express-v3": "npm:apollo-server-express@^3.6.2",
+    "apollo-server-express": "^3.13.0",
```

Comment:
Q: Why dont we use https://github.com/apollographql/apollo-server/tree/main/packages/server?